scripts/bash/validate-frontmatter.sh

```
@@ -0,0 +1,150 @@
+    if [[ "$created_date" =~ ^[0-9]{4}-[0-9]{2}-[0-9]{2}$ ]]; then
+        echo "[INFO] Date format valid: $created_date"
+    else
+        echo "[WARN] Date format may be invalid: $created_date (expected: YYYY-MM-DD)"
+    fi
+fi
+
+# Validate status field if present
+status=$(echo "$frontmatter" | grep -oP 'status:\s*\K\S+' || echo "")
+if [ -n "$status" ]; then
+    valid_statuses=("draft" "in-progress" "review" "approved" "implemented" "archived")
+    if [[ " ${valid_statuses[@]} " =~ " ${status} " ]]; then
```

Shellcheck (0.11.0)

[error] Arrays implicitly concatenate in [[ ]]. Use a loop (or explicit * instead of @).

(SC2199)